src/pages/ClaimsPage/ClaimsPage.tsx

```diff
@@ -126,22 +108,38 @@ export default function ClaimsPage() {
             { title: 'Epochs', css: styles.th },
             { title: 'Rewards', css: styles.thLast },
           ]}
-          data={currentClaims(claims.filter(c => !c.txHash))}
+          data={unclaimedClaimsRows}
           startingSortIndex={2}
-          startingSortDesc
+          startingSortDesc={false}
```

Comment:
weirdly: this seems to sort by newest at the top

Reply:
In order for sorting to work in this table component, you have to define `sortByColumn`. It's a function that takes a column index and returns a function that produces a value to sort by. Right now it's just a placeholder `() => c => c` which means "no matter what column this is, just sort by the entire data object for each row". You can see a working example [here](https://github.com/coordinape/coordinape/blob/main/src/pages/DistributionsPage/AllocationsTable.tsx#L113)